commands/run/run.go

```diff
@@ -180,7 +180,13 @@ func (app *App) Run() {
 		}
 	}
 	// Build the app.
-	command := fmt.Sprintf(`go build -o %s %s %s`, outputPath, app.Options, app.File)
+	var command string
+	if app.Options == "" {
+		command = fmt.Sprintf(`go build -o %s %s`, outputPath, app.File)
+	} else {
+		command = fmt.Sprintf(`go build -o %s %s %s`, outputPath, app.Options, app.File)
```

Comment:
If the `app.Options` is empty, then it is the same as `fmt.Sprintf(`go build -o %s %s`, outputPath, app.File)`.

Reply:
```go
package main

import (
	"fmt"
)

func main() {
	outputPath := "/tmp/gf-cli/main"
	appOptions := ""
	appFile := "main.go"
	command1 := fmt.Sprintf(`go build -o %s %s %s`, outputPath, appOptions, appFile)
	command2 := fmt.Sprintf(`go build -o %s %s`, outputPath, appFile)
	fmt.Println(command1==command2)
	// false
}

```

Reply:
There's some extra blanks in the command line if `appOptions` is empty but it does not affect its functionality.